unison-core/src/Unison/Reference.hs

```diff
@@ -76,6 +77,10 @@ pattern Derived h i = DerivedId (Id h i)
 _DerivedId :: Prism' Reference Id
 _DerivedId = _Ctor @"DerivedId"
 
+isBuiltin :: Reference -> Bool
```

Comment:
I didn't end up using this I don't think, but seems worth having (it exists in the V1 package)